### PR TITLE
Reorganize utils

### PR DESCRIPTION
**Pull request by mindrones**

### CODE REVIEW

**Review by saabi — APPROVED** [review]
Finally much saner, nice work!

File: src/node_modules/app/utils/generic.js [hunk 1]
```diff
@@ -9,3 +9,19 @@ let lastId = 0;
 export function autoID () {
 	return `${IS_BROWSER ? 'client' : 'server'}${lastId++}`;
 }
+
+// TODO -> evaluateIsFunction
```
Comment (saabi) [inline src/node_modules/app/utils/generic.js]: ttypo

Comment (mindrones) [reply src/node_modules/app/utils/generic.js]: was fixed already

**Review by saabi — APPROVED** [review]
It's shinier :)

**Pull request merged**